From 4b85ae6fef0bbf7f17d2a02a3c841378c2cf337e Mon Sep 17 00:00:00 2001
From: Diab Neiroukh <lazerl0rd@thezest.dev>
Date: Tue, 26 Jan 2021 12:29:17 +0000
Subject: [PATCH] x86: Use the correct header for config descriptions.

---help--- is no longer supported by upstream, so we replace their usage
with help.

Signed-off-by: Diab Neiroukh <lazerl0rd@thezest.dev>
---
 arch/x86/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/x86/Kconfig b/arch/x86/Kconfig
index c0eebbbe5555..e3c59118ab94 100644
--- a/arch/x86/Kconfig
+++ b/arch/x86/Kconfig
@@ -1163,14 +1163,14 @@ config X86_THERMAL_VECTOR
 	def_bool y
 	prompt "Machine check thermal vector"
 	depends on X86_MCE_INTEL
-	---help---
+	help
 	  Provide support for capturing thermal events, logging them, and
 	  passing them off to other drivers.
 
 config X86_MCE_THERMAL_VERBOSE
 	bool "Verbose logging for thermal events"
 	depends on X86_THERMAL_VECTOR
-	---help---
+	help
 	  Display messages in the kernel log when thermal events are triggered,
 	  such as overheating and throttling. This mostly only uses extra CPU
 	  for ratelimiting and book keeping, so unless you need these logs, it
